TESTING.md

```diff
@@ -0,0 +1,46 @@
+# Cloud ML Samples testing
+
+This repository uses a Google service Kokoro for CI testing. The following guide covers how to modify existing tests, and write new tests.
+
+## Running Tests
+
+All presubmits will automatically be run on all pull requests which are made by a repository administrator, or have the label `kokoro:run`.
+
+For Ubuntu for example a presubmit will then chain together the following scripts:
+1. [`testing/ubuntu/presubmit/run_if_changed.sh`](testing/ubuntu/presubmit/run_if_changed.sh) will exit if there are no changes in `${CMLE_TEST_BASE_DIR}`, otherwise it will run
+2. [`testing/ubuntu/setup_and_run_test.sh`](testing/ubuntu/setup_and_run_test.sh) which performs setup common to our examples, and then runs the test specified by
+3. `${CMLE_TEST_SCRIPT}` which is specified in the build config file defining the test (for example [`testing/ubuntu/presubmit/flowers.cfg`](testing/ubuntu/presubmit/flowers.cfg)
+
+Tests past if the script chain exits with status 0.
+
+### Getting Logs Access
+
+Logs are provided through Stackdriver logging and the Cloud Console. Frequent contributors or contributors of complex examples may want to get Log access for the project `cloudml-samples-testbed` which will allow them to follow through on the links.
```

Comment:
@elibixby Can you grant me and @nkashy1 access to cloudml-samples-testbed. Also I think this can be a friction point since we are proposing external contributors will need access to cloudml-samples-testbed. How will that work?